kola/tests/misc/selinux.go

```diff
@@ -17,52 +17,150 @@ package misc
 import (
 	"github.com/coreos/mantle/kola/cluster"
 	"github.com/coreos/mantle/kola/register"
+	"github.com/coreos/mantle/platform"
+	"regexp"
```

Comment:
We structure imports in the following order (with an extra line break between categories): stdlib, external, internal

```golang
import (
    "regexp"

    "github.com/coreos/mantle/kola/cluster"
    "github.com/coreos/mantle/kola/register"
    "github.com/coreos/mantle/platform"
)
```